src/engine/server_iv.c

```diff
@@ -797,6 +797,15 @@ ds_iv_ns_start(struct ds_iv_ns *ns)
 	ds_iv_ns_get(ns);
 }
 
+void
+ds_iv_ns_leader_stop(struct ds_iv_ns *ns)
+{
+	/* Set iv_stop on the leader, so all arriving IV requests will return
```

Comment:
Is "the leader" the same concept as "the master rank"?

Reply:
yes, it is the same. sorry, probably need unify the name here.